templates/index.html

```diff
@@ -1,18 +1,73 @@
 {% extends "layout.html" %}
 {% block title %}To-Do App{% endblock %}
-
 {% block content %}
   <div class="jumbotron">
     <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <p class="lead">Toby's To Do List.</p>
   </div>
-
-  <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+    <div class="col-12-xs">
+      <table class="table table-borderless table-striped">
+        <thead>
+          <tr>
+            <th>ID</th>
+            <th>Task Title</th>
+            <th>Task Status</th>
+            <th></th>
+            <th></th>
+          </tr>
+        </thead>
+        <tbody>
+          {% for todo in todos %}
+            <tr>
+              <td>
+                {{ todo.id }}
+              </td>
+              <form action="/update-todo" method="post">
+                <input type="hidden" name="todo_id" value="{{ todo.id }}">
+                <td>  
+                    <input type="text" name="title" value="{{ todo.title }}" class="form-control" aria-label="Default" aria-describedby="inputGroup-sizing-default">
+                </td>
+                <td>
+                    <select name="status" id="status" class="custom-select" id="inputGroupSelect01">
```

Comment:
You don't need an id and actually this id is bad practice because you are including multiple elements on the page with the same id.